compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangDocumentationAttribute.java

```diff
@@ -0,0 +1,72 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+package org.wso2.ballerinalang.compiler.tree.expressions;
+
+import org.ballerinalang.model.tree.IdentifierNode;
+import org.ballerinalang.model.tree.NodeKind;
+import org.ballerinalang.model.tree.expressions.DocumentationAttributeNode;
+import org.ballerinalang.model.tree.expressions.ExpressionNode;
+import org.wso2.ballerinalang.compiler.tree.BLangIdentifier;
+import org.wso2.ballerinalang.compiler.tree.BLangNodeVisitor;
+
+/**
+ * @since 0.94
+ */
+public class BLangDocumentationAttribute extends
+        BLangExpression implements DocumentationAttributeNode {
+
+    public BLangIdentifier documentationField;
+    public BLangExpression documentationText;
+
+    @Override
+    public BLangIdentifier getDocumentationField() {
+        return documentationField;
+    }
+
+    @Override
+    public void setDocumentationField(IdentifierNode field) {
+        this.documentationField = (BLangIdentifier) field;
+    }
+
+    @Override
+    public BLangExpression getDocumentationText() {
+        return documentationText;
+    }
+
+    @Override
+    public void setDocumentationText(ExpressionNode documentationText) {
+        this.documentationText = (BLangExpression) documentationText;
+    }
+
+    @Override
+    public void accept(BLangNodeVisitor visitor) {
+        visitor.visit(this);
+    }
+
+    @Override
+    public NodeKind getKind() {
+        return NodeKind.DOCUMENTATION_ATTRIBUTE;
+    }
+
+    @Override
+    public String toString() {
+        return "BLangDocAttachmentAttribute: " + documentationField != null ? documentationText.toString() + " : "
```

Comment:
According to the operator precedence rules : above will check whether the below expression is true or false. (Which is resulted in true always)
`"BLangDocAttachmentAttribute: " + documentationField != null`

Reply:
+1